cost/google/cloud_sql_idle_instance_recommendations/google_sql_idle_instance_recommendations.pt

```diff
@@ -0,0 +1,450 @@
+name "Google Cloud SQL Idle Instance Recommender"
+rs_pt_ver 20180301
+type "policy"
+short_description "This Policy finds Google Idle Cloud SQL Instance Recommendations and reports when it finds them. See the [README](https://github.com/flexera/policy_templates/tree/master/cost/google/recommender) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
```

Comment:
As per other comment - would have thought the first link would need to match the path of the readme which is `cost/google/cloud_sql_idle_instance_recommendations`

2nd link should be the flexera automation link rather than the rightscale one:  `[docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more.`